docs/funcx_upgrade.rst

```diff
@@ -0,0 +1,19 @@
+###########################################
+Upgrading from funcX SDK and funcx-endpoint
+###########################################
+
+Background
+^^^^^^^^^^
+
+Placeholder for background text of rebranding funcX as Globus Compute
+
+funcX SDK
+^^^^^^^^^
+
+Placeholder for detailed steps and notes when upgrading the SDK.
+
+funcX Endpoint
+^^^^^^^^^^^^^^
+
+Placeholder for detailed steps and notes when upgrading the endpoint.
+
```

Comment:
TODO yet?

Reply:
I intended it to be a separate story along with a changelog.  For now, I'll put something minimal here and wait til this PR is merged before fleshing it out.